index.html

```diff
@@ -13,108 +13,117 @@
-         <a href="https://fosstodon.org/@tldr_pages" target="_blank"><img alt="Fosstodon" src="https://img.shields.io/badge/fosstodon-tldr__pages-blue"></a>
-         
-        <a href="https://matrix.to/#/#tldr-pages:matrix.org" target="_blank"><img alt="Matrix" src="https://img.shields.io/matrix/tldr-pages:matrix.org?label=chat+on+Matrix"></a>
-      </p>
-
-      <p>
-        The tldr pages are a community effort
-        to simplify the beloved <a href="https://en.wikipedia.org/wiki/Man_page">man pages</a>
-        with practical examples.
-      </p>
-      <p><img src="assets/img/screenshot.png" alt="tldr screenshot"></p>
-
-      <p>
-        Try the <a href="https://tldr.inbrowser.app" target="blank">live demo</a> below,
-        have a look at the <a href="/assets/tldr-book.pdf">PDF version</a>,
-        or follow the <a href="#installation">installation instructions</a>.
-      </p>
-
-      <noscript>
-        <div id="enable_js">
-          <p>The live demo requires JavaScript to function. To access the content, please enable JavaScript in your browser settings.</p>
-        </div>
-      </noscript>
-      
-      <iframe id="iframe-content" src="https://tldr.inbrowser.app/pages/common/tar" width="100%" height="500px" style="border: 1px solid"></iframe>
-
-      <h2 id="installation">
-        <a href="#installation">§</a>
-        Installation
-      </h2>
-
-      <p>As of now, our most mature client is the Node.js one, which you can easily install from npm:</p>
-      <pre>npm install -g tldr</pre>
-
-      <p>
-        There are many more clients that have sprouted from the community, including:
+        <h1>tldr pages</h1>
+        <img src="Moon.png" alt="" id="icon">
+        <blockquote>Simplified and community-driven man pages</blockquote>
+        <p class="widgets">
+            <a href="https://github.com/tldr-pages/tldr" target="_blank"><img alt="GitHub stars" src="https://img.shields.io/github/stars/tldr-pages/tldr.svg?logo=github"></a>
+            <a href="https://twitter.com/tldr_pages" target="_blank"><img alt="Twitter" src="https://img.shields.io/badge/twitter-tldr__pages-%231DA1F2.svg"></a>
+            <a href="https://fosstodon.org/@tldr_pages" target="_blank"><img alt="Fosstodon" src="https://img.shields.io/badge/fosstodon-tldr__pages-blue"></a>
+            <a href="https://matrix.to/#/#tldr-pages:matrix.org" target="_blank"><img alt="Matrix" src="https://img.shields.io/matrix/tldr-pages:matrix.org?label=chat+on+Matrix"></a>
+        </p>
+        <p>
+            The tldr pages are a community effort
+            to simplify the beloved <a href="https://en.wikipedia.org/wiki/Man_page">man pages</a>
+            with practical examples.
+        </p>
+        <p><img src="assets/img/screenshot.png" alt="tldr screenshot"></p>
+        <p>
+            Try the <a href="https://tldr.inbrowser.app" target="blank">live demo</a> below,
+            have a look at the <a href="/assets/tldr-book.pdf">PDF version</a>,
```

Comment:
```suggestion
            have a look at the <a href="https://github.com/tldr-pages/tldr/releases/latest/download/tldr-book.pdf">PDF version</a>,
```
We are currently migrating assets from this repo to GitHub releases (see https://github.com/tldr-pages/tldr/issues/12048) so IG we can update this link to future proof it.